github-status/handler.go

```diff
@@ -33,8 +33,7 @@ var (
 
 // Handle a serverless request
 func Handle(req []byte) string {
-
-	if hmacEnabled() {
```

Comment:
I preferred the old name because it's more descriptive and abstracts away the env-var. Can you revert please?

Reply:
Yep no problems